week-4-practise/level-2/CODEWARS.md

```diff
@@ -3,4 +3,33 @@ Complete the following CodeWars exercises. Go to the webpages below and follow t
 Click "ATTEMPT" to test your solution.
 
 Exercises:
-- https://www.codewars.com/kata/welcome/train/javascript
\ No newline at end of file
+
+- https://www.codewars.com/kata/welcome/train/javascript
+
+var audience = {
+english: 'Welcome',
+czech: 'Vitejte',
+danish: 'Velkomst',
+dutch: 'Welkom',
+estonian: 'Tere tulemast',
+finnish: 'Tervetuloa',
+flemish: 'Welgekomen',
+french: 'Bienvenue',
+german: 'Willkommen',
+irish: 'Failte',
+italian: 'Benvenuto',
+latvian: 'Gaidits',
+lithuanian: 'Laukiamas',
+polish: 'Witamy',
+spanish: 'Bienvenido',
+swedish: 'Valkommen',
+welsh: 'Croeso'
+};
+
+function greet(language) {
+if (audience.hasOwnProperty(language)) {

```

Comment:
As above, don't access variables from outside of the function that aren't passed in as an argument. Instead, you could define the `audience` variable inside of `greet()`.